packages/b/bchaddrjs.json

```diff
@@ -13,6 +13,7 @@
     "bitpay",
     "cash-address"
   ],
+  "filename": "bchaddrjs-0.4.9.min.js",
```

Comment:
```suggestion
```
I don't think we should set a default file here when it changes with each version -- if a new version is pulled in this will become incorrect and will be misleading. There is no default file for this library